docusaurus/docs/recipes/splash-screen.md

```diff
@@ -0,0 +1,98 @@
+---
+id: splash-screen
+title: Adding a splash screen
+sidebar_label: Splash screen
+---
+
+Certain pages have impactful experiences. These experiences can make the total bundle size larger as they pack possibly large dependencies and media assets, such as 3D objects and audio files.
+
+It's then often normal to preload all the required files for an uninterrupted experience. This recipe explains how you can leverage [`@moxy/react-wait-for-react`](https://github.com/moxystudio/react-wait-for-react) to display a **splash screen** before your static or server-side rendered app becomes interactive, and optionally until all the required files are loaded (via a promise).
+
+⚠️ You should still render the app or page contents "below" the splash screen, to keep your website SEO friendly.
+
+## Walk-through
+
+### 1. Install `@moxy/react-wait-for-react`
+
+```sh
+npm i `@moxy/react-wait-for-react`
+```
+
+### 2. Add `<SplashScreen />` to your project
+
+Copy the [`splash-screen`](https://github.com/moxystudio/next-with-moxy/tree/master/docusaurus/static/recipes-assets/splash-screen) component folder into `www/shared/components` and edit `www/shared/components/index.js` to export it:
+
+```js
+// ....
+export { default as SplashScreen } from './splash-screen';
+```
+
+Finally, adjust the component you just copied according to what was designed for your project.
+
+ℹ️ As a rule of thumb, the progress / loading indicator should only be shown after **2 or 3 seconds**.
```

Comment:
Why is this? I think we should add a small explanation for this rationale.

Reply:
The reason is to avoid showing a loader right at the start, when the preload is usually fast on a good connection. We use this strategy in moxy.studio, where we show a welcome message and only a loader if it's taking too long.

Splash pages on mobile act the same, they show an app icon and only after a few seconds they might show some loading indicator.

Reply:
I've improved the sentence, can you check if it's better?

Reply:
It is, resolving!